projects/cashmere/src/lib/table/bind-observable/bind-observable.spec.ts

```diff
@@ -0,0 +1,271 @@
+import {BindObservable} from './bind-observable';
+import {Observable} from 'rxjs';
+import {take} from 'rxjs/operators';
+
+function WrappingDecorator(): PropertyDecorator {
+    return (target: any, propertyName: string | symbol) => {
```

Comment:
I have no idea what a `symbol` type is even after looking at the docs. How is it used exactly?